images/win/scripts/Installers/Install-Swig.ps1

```diff
@@ -0,0 +1,8 @@
+################################################################################
+##  File:  Install-Swig.ps1
+##  Desc:  Install Swig
+################################################################################
+
+Choco-Install -PackageName swig
+
+Invoke-PesterTests -TestFile "Tools" -TestName "swig"
```

Comment:
Test name starts with the capital letter
```suggestion
Invoke-PesterTests -TestFile "Tools" -TestName "Swig"
```

Reply:
Fixed